docs/CHILI-GraFx/users/intro/index.md

```diff
@@ -1,79 +1,78 @@
-# User management
+# User Management
 
 ## Introduction
 
-4 basic concepts define the user management. Below you can find the details of each concept.
+Five basic concepts define user management. Below you can find the details of each concept.
 
-There are 4 different concepts that make up the user management process in CHILI GraFx.
+There are four different concepts that make up the user management process in CHILI GraFx:
 
-	- Resources
-	- Permissions
-	- Roles
-	- CHILI GraFx Users
+- Resources
+- Access
+- Roles
+- CHILI GraFx Users
+- User Groups
 
-Let's dive in to each concept below to see what they do, and how they relate.
+Let's dive into each concept below to see what they do, and how they relate.
 
 ### Resources
 
 !!! Definition
-	Any item that will be impacted by the user management system.
+    Any item that will be impacted by the user management system.
 
-Resources are items that make up CHILI GraFx. Control and management of these resources require the proper _permissions_
+Resources are items that make up CHILI GraFx. Control and management of these resources require the proper _access_.
 
 **Example resources:**
 
 GraFx Media assets, GraFx Templates, Users, Environments
```

Comment:
maybe add as example: Collections in GraFx Studio
(because user access management of collections is the next update we'll bring)